### PR TITLE
`restart: always` in Docker Swarm example.

### PR DESCRIPTION
**Pull request by alicethorne-ab**
Fixes another issue with Docker Swarm not restarting containers on reboot.

### CODE REVIEW

**Review by accraw — APPROVED** [review]
Looks good!

File: docker/swarm/docker-compose.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ services:
     deploy:
       replicas: 1
       restart_policy:
-        condition: on-failure
+        condition: always
```
Comment (devillexio) [inline docker/swarm/docker-compose.yml:8]: Is `always` the correct value or should it be `any`?

I'm not that familiar with Docker Swarm, but it seems to follow the docker-compose file conventions. According to the [docker-compose docs](https://docs.docker.com/compose/compose-file/compose-file-v3/) these are the valid values for `restart_poilcy.condition`:

image

Comment (alicethorne-ab) [reply docker/swarm/docker-compose.yml:8]: You are right, unfortunately. Apparently I hadn't tested the correct mode here. I'll open a new PR to fix this bug.

Comment (devillexio) [reply docker/swarm/docker-compose.yml:8]: My apologies for not getting to the review sooner. 😞

**Pull request merged**